CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.11.0] - 2024-01-03
+### New
+- `User.aiDailyMessageLimit` to determine how many AI image description requests an Explorer can make per day (#102)
+- `User.getAIDailyMessageLimitProperty` to retrieve property from `UserProperties` (#102)
```

Comment:
This is a private method, so you don't need to mention it in the changelog.

Reply:
Okay, I'll remove it